static/js/app.js

```diff
@@ -3,106 +3,158 @@
 
-var textArea = document.querySelector('#doc_view');
+var textArea = document.querySelector('#doc-view');
 var currentDocument = new Doc("test", testContent);
-var currentHighlighterColor;
-// future implementation
+
+//default assign first label
+var firstLabel = $('.label')[0];                                      //testing purposes remove when done
+var currentLabel = firstLabel.getAttribute('value');                  //remove assignment when done
+$(firstLabel).attr('id', 'label-selected');                           //remove when done
+delete firstLabel;                                                    //remove when done
+
+//future implementation
 var openDocuments = [];
-var currentLabel;
+openDocuments.push(currentDocument);
 
 //download highlights
-$("button").click(function () {
-    if (this.target.id === 'export') {
-        if (openDocuments.length === 0) {
-            alert("Error: No data to download!");
-            return;
-        }
-        console.log(JSON.stringify(openDocuments));
+$('button').click(function () {
+  if (this.id === 'export') {
+    if (openDocuments.length === 0) {
+      alert("Error: No data to download!");
+      return;
     }
+
+    console.log(JSON.stringify(openDocuments));
+  }
 });
+console.log(currentLabel);                                            //remove when done
+
+//testing purposes //remove when implementing dynamic adding 
+{
+  $('head').append(
+    $('<style/>', {
+      id: 'lorem-style',
+      html: '.hwt-content .label_lorem {background-color: rgb(255, 0, 0);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'dolor-style',
+      html: '.hwt-content .label_dolor {background-color: rgb(255, 255, 0);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'rem-style',
+      html: '.hwt-content .label_rem {background-color: rgb(255, 192, 203);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'sit-style',
+      html: '.hwt-content .label_sit {background-color: rgb(173, 255, 47);}'
+    })
+  );
+}
```

Comment:
You should put the selector in a variable to improve performance. See: https://codereview.stackexchange.com/questions/70298/jquery-duplicated-selector

Reply:
updated for better performance + less redundancy